samples/excel/range/copy-move-multiply-values.yaml

```diff
@@ -0,0 +1,276 @@
+        let context = <Excel.RequestContext>sheet.context;
+
+        sheet.load("name, position");
+
+        await context.sync();
+
+        let newSheet = context.workbook.worksheets.add();
+        newSheet.position = sheet.position;
+        sheet.delete();
+        newSheet.name = sheet.name;
+        newSheet.activate();
+
+        await context.sync();
+    }
+
+    async function copyValues() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                let fromRange = sheet.getRange("B2:E5");
+                fromRange.load("values");
+
+                await context.sync();
+                
+                let toRange = sheet.getRange("B10:E13");
+                toRange.values = fromRange.values;
+
+                await context.sync();
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+    async function moveValues() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                let fromRange = sheet.getRange("B2:E5");
+                fromRange.load("values");
+
+                await context.sync();
+
+                let toRange = sheet.getRange("B10:E13");
+                toRange.values = fromRange.values;
+
+                fromRange.clear(Excel.ClearApplyTo.all);
+
+                await context.sync();
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+    async function multiplyValuesForLoop() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                let range = sheet.getRange("C3:C5");
+                range.load("values");
+
+                await context.sync();
+
+                var vals = range.values;
+                for (var i = 0; i < vals.length; i++){
+                    for (var j = 0; j < vals[i].length; j++){
+                        vals[i][j] = vals[i][j] * 10;
+                    }
+                }
+                range.values = vals;
+
+                range.format.autofitColumns();
+                
+                await context.sync();
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+    async function multiplyValuesMap() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                let range = sheet.getRange("C3:C5");
+                range.load("values");
+
+                await context.sync();
+
+                range.values = range.values.map((rowValues) => {
```

Comment:
You're mixing lambda and return syntax.

This could also be done as:

range.values = range.values.map(
    rowValues => rowValues.map(cellValue => cellValue * 10)
);

Or if that doesn't look clean enough for you, a slightly more verbose version (but at least self-consistent at each level):

range.values = range.values
	.map(rowValues => {
	    return rowValues.map(cellValue => cellValue * 10);
	});